### ISSUE TITLE
The minimum requirement version of Perl may be 5.10.0

### ISSUE THREAD

**Issue by twata1** [Leont/cpan-requirements-dynamic#1]
Hello,

CPAN::Requirements::Dynamic depends on [CPAN::Meta::Requirements::Range](https://metacpan.org/pod/CPAN::Meta::Requirements::Range), but its Perl minimum requirement version seems to be 5.10.0.

P.S.
So I think the minimum required Perl version for Dist::Build will also be 5.10.0.

Thank you,

**Comment by Leont** [comment]
Yeah, that's because the minimum version of CPAN::Meta::Requirements got bumped from 5.8 to 5.10.
